beamerinnerthemeusyd.sty

```diff
@@ -57,9 +58,15 @@
 \defbeamertemplate*{logo titlepage}{usyd}
 {\includegraphics[width=2.4cm]{\USYD@logo}}
 
+\defbeamertemplate*{affiliationlogo titlepage}{usyd}
+{\includegraphics[width=2.2cm]{\USYD@affiliationlogo}}
+
 \defbeamertemplate*{logo}{usyd}
 {\includegraphics[height=3em]{\USYD@logo}}
 
+\defbeamertemplate*{affiliationlogo}{usyd}
+{\includegraphics[height=3em]{\USYD@affiliationlogo}}
```

Comment:
This isn't currently used anywhere, the logo template is used for the logobar theme. I'm not sure that having two logos on every slide is a good idea. Should it be used?